### PR TITLE
separate trusted and native llm cpu finetune from lora

### PR DESCRIPTION
**Pull request by Uxito-Ada**
## Description

a refactor is needed to separate bigdl lora finetuning(with TDX) into two, trusted-llm and native finetuning without TDX respectively. And we aim to provide both CPU and XPU versions and now start with CPU one.

### 1. Why the change?

as above

### 2. User API changes

no

### 3. Summary of the change 

separate trusted-llm and bigdl from lora finetuning

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

**Comment by Uxito-Ada** [comment]
TODO: append trusted-llm k8s readme and codes

File: docker/llm/finetune/lora/cpu/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+ARG HTTP_PROXY
+ARG HTTPS_PROXY
+
```
Comment (glorysdj) [inline docker/llm/finetune/lora/cpu/docker/Dockerfile:3]: use  PIP_NO_CACHE_DIR=false to turn off pip cache and reduce the size of the image
https://github.com/intel-analytics/BigDL/blob/main/docker/llm/inference/cpu/docker/Dockerfile#L5

File: docker/llm/finetune/lora/cpu/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+FROM mpioperator/intel as builder
+
+ARG HTTP_PROXY
+ARG HTTPS_PROXY
+ADD ./requirements.txt /ppml/requirements.txt
+
+RUN mkdir /ppml/data && mkdir /ppml/model && mkdir /ppml/output && \
+# install pytorch 2.0.1
+    export http_proxy=$HTTP_PROXY && \
```
Comment (glorysdj) [inline docker/llm/finetune/lora/cpu/docker/Dockerfile:12]: use arg http_proxy/https_proxy and let user to choose if proxy is needed to set

**Review by xiangyuT — APPROVED** [review]
LGTM

**Pull request merged**